backend/twitterbot/management/commands/load_tweetinfo.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+
+from tournesol.entities.video import YOUTUBE_UID_NAMESPACE
+from tournesol.models.entity import Entity
+from twitterbot.models.tweeted import TweetInfo
+
+
+class Command(BaseCommand):
+    help = "Load already tweeted videos from a file and save data as TweetInfo."
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "-f",
+            "--file",
+            type=str,
+            help="Relative path to the input file.",
```

Comment:
It could be useful to add some details about the format of this file in this help message.

Reply:
:+1: Something like this?

```
help="Relative path to the input file. Each line must follow the structure {yt_video_id};{BOT_NAME}.",

Reply:
:+1: Something like this?

```
help="Relative path to the input file. Each line must follow the structure {yt_video_id};{BOT_NAME}.",